pep-0517.txt

```diff
@@ -307,6 +313,10 @@ Of course, it's up to users to make sure that they pass options which
 make sense for the particular build backend and package that they are
 building.
 
+The hooks may be called with positional or keyword arguments, so backends
```

Comment:
This is again, a total bikeshed, but maybe we should mandate that these are always keyword arguments? That makes it easier to add new, optional arguments in the future.